Nikita_Korzhavin/api/orderAPI.js

```diff
@@ -0,0 +1,46 @@
+var ProductFactory = require('./productFactory.js');
+
+// просто вынесенные функции из order.js
```

Comment:
Well, actually I don't think it's a good idea to have all these functions in one file here.
`isNeededObject` or `arrayWithDeletedProduct` are seem to be auxiliary function for general purposes
`calculateOrder...` or `payOrder` is better to store inside `Order`

Reply:
fixed